sklearn/ensemble/gradient_boosting.py

```diff
@@ -953,6 +960,8 @@ def fit(self, X, y, sample_weight=None, monitor=None):
             computing held-out estimates, early stopping, model introspect, and
             snapshoting.
 
+        n_features_ : the number of features.
```

Comment:
This should appear in an Attributes section of the estimator docstrings, not the Parameters section of the `fit` docstring.

However, I'm still tempted to say it should be a private `_n_features`, not a public `n_features_`